execution/executor-benchmark/src/main.rs

```diff
@@ -202,17 +207,12 @@ struct ProfilerOpt {
     memory_profiling: bool,
 }
 
-#[derive(Parser, Debug)]
-#[clap(group(
-    ArgGroup::new("vm_selection")
-    .args(&["use_native_executor", "use_ptx_executor"]),
-))]
-pub struct VmSelectionOpt {
-    #[clap(long)]
-    use_native_executor: bool,
-
-    #[clap(long)]
-    use_ptx_executor: bool,
+#[derive(Parser, Debug, ValueEnum, Clone, Default)]
+enum BlockExecutorTypeOpt {
+    #[default]
+    AptosVMWithBlockSTM,
+    NativeLooseSpeculative,
```

Comment:
I have to say the meaning of "speculative" isn't clear here, and I don't have a better suggestion for now -_-

Reply:
I'll add the comments to all variants here, but I was surprised to see that current implementation was replacing BlockSTM as well, not just AptosVM - so wanted to be more clearer in name.

confusing name is better than name that suggests wrong thing - as folks will ask around if they don't understand it :)

but open to all name suggestions. These are all the names:
```
     AptosVMWithBlockSTM,
     NativeVMWithBlockSTM,
     NativeLooseSpeculative, 
     NativeValueCacheLooseSpeculative,
     NativeNoStorageLooseSpeculative,
```

Reply:
Please please please add documentation to all variants here: it is not clear at all without context what are those. Also, as a nit:
`BlockSTMWith...VM` reads better? Block executor which uses some particular VM impl?